### PR TITLE
RAD-102 Change spring config to annotation-driven

### PR DESCRIPTION
**Pull request by bgeVam**
see https://issues.openmrs.org/browse/RAD-102

### CODE REVIEW

File: omod/src/test/java/org/openmrs/module/radiology/web/controller/PortletsControllerTest.java [hunk 1]
```diff
@@ -49,6 +49,7 @@
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 import org.springframework.beans.TypeMismatchException;
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (teleivo) [inline omod/src/test/java/org/openmrs/module/radiology/web/controller/PortletsControllerTest.java]: this one is never used, please remove

File: omod/src/test/java/org/openmrs/module/radiology/web/controller/RadiologyObsFormControllerTest.java [hunk 2]
```diff
@@ -73,13 +69,7 @@
...
 	
 	@Mock
-	private OrderService orderService;
-	
-	@Mock
-	private EncounterService encounterService;
-	
-	@Mock
-	private AdministrationService administrationService;
+	RadiologyProperties radiologyProperties;
```
Comment (teleivo) [inline omod/src/test/java/org/openmrs/module/radiology/web/controller/RadiologyObsFormControllerTest.java]: why not private like others?

File: omod/src/test/java/org/openmrs/module/radiology/web/controller/PortletsControllerTest.java (hunk 1 above)
Comment (bgeVam) [inline omod/src/test/java/org/openmrs/module/radiology/web/controller/PortletsControllerTest.java]: check

File: omod/src/test/java/org/openmrs/module/radiology/web/controller/RadiologyObsFormControllerTest.java (hunk 2 above)
Comment (bgeVam) [inline omod/src/test/java/org/openmrs/module/radiology/web/controller/RadiologyObsFormControllerTest.java]: check

**Pull request closed without merging**